ccd-definition/CaseEvent/Camunda/CaseFlagsEvents-nonprod.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I think this wouldn't be excluded since the filenames have to be *CaseFlags-nonprod.json